tests/snapshots/tests.contrib.django.test_django_snapshots.test_asgi_200[application2].json

```diff
@@ -18,18 +18,18 @@
       "http.status_code": "200",
       "http.url": "http://localhost:8000/",
       "http.version": "1.1",
-      "runtime-id": "09c0f53e3e5748c29ff8f04cad464bac"
+      "runtime-id": "31e2968631db495a8fe79e021980080a"
```

Comment:
I am not sure I follow what is changing in these fixtures, they are being renamed, but they are the same as they were previously, do we need `[application]`/`[application2]` in the name?

Reply:
Nope. I was going to looking to creating one snapshot for both parameters.